tests/Type/EnumTypeTest.php

```diff
@@ -655,4 +655,57 @@ public function testCallsOverwrittenEnumTypeMethods(): void
             GraphQL::executeQuery($this->schema, $query, null, null, $variables)->toArray(DebugFlag::INCLUDE_DEBUG_MESSAGE)
         );
     }
+
+    public function testLazilyDefineValuesAsCallable(): void
+    {
+        $called = false;
+        $ColorType = new EnumType([
+            'name'   => 'Color',
+            'values' => function() use (&$called) : iterable {
+                $called = true;
+                yield 'RED'   => ['value' => 0];
+                yield 'GREEN' => ['value' => 1];
+                yield 'BLUE'  => ['value' => 2];
```

Comment:
```suggestion
                yield 'RED'   => ['value' => 0];
```

Should suffice for this test